Source/CarthageKit/VersionFile.swift

```diff
@@ -49,7 +49,13 @@ struct VersionFile: Codable {
 		}
 	}
 
-	init(commitish: String, macOS: [CachedFramework]?, iOS: [CachedFramework]?, watchOS: [CachedFramework]?, tvOS: [CachedFramework]?) {
+	init(
+		commitish: String,
+		macOS: [CachedFramework]?,
+		iOS: [CachedFramework]?,
+		watchOS: [CachedFramework]?,
+		tvOS: [CachedFramework]?
+	) {
```

Comment:
Are there any functional changes in this file? I'm having trouble spotting anything with the reformatting.

Reply:
No there are no functional changes.